content/technical/components.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "R-hub components"
```

Comment:
each page you had should have a title and some content.

for images https://gohugo.io/content-management/shortcodes/#figure